### PR TITLE
Update short_description.txt

### PR DESCRIPTION
**Pull request by gallegonovato**

### CODE REVIEW

File: fastlane/metadata/android/es-ES/short_description.txt [hunk 1]
```diff
@@ -1 +1 @@
-Configuración individual de volumen de voz y música para dispositivos Bluetooth.
\ No newline at end of file
+Configuración individual del volumen de la voz y la música para los dispositivos Bluetooth.
```
Comment (d4rken) [inline fastlane/metadata/android/es-ES/short_description.txt:1]: The text can only be 80 characters long, is it possible to make it shorter?

**Review by d4rken — CHANGES_REQUESTED** [review]

File: fastlane/metadata/android/es-ES/short_description.txt (hunk 1 above)
Comment (gallegonovato) [reply fastlane/metadata/android/es-ES/short_description.txt:1]: Configuración separada del volumen de voz y música para dispositivos Bluetooth

Configura el volumen individual de voz y música para dispositivos Bluetooth

Administra el volumen para la música y las llamadas en dispositivos Bluetooth.

Comment (d4rken) [reply fastlane/metadata/android/es-ES/short_description.txt:1]: I don't understand Spanish, I trust your judgement, select something you like :slightly_smiling_face:.

Comment (gallegonovato) [reply fastlane/metadata/android/es-ES/short_description.txt:1]: Good afternoon @d4rken

Sorry for not telling you which one I would put.

I would put:

Administra el volumen para la música y las llamadas en dispositivos Bluetooth.

Sorry for the mistake of not telling you which one fits best. I don't know where I left my head...


I apologize again, it seems that my neuron went on a weekend or a party.


A hug

**Pull request merged**